server/router_utils.go

```diff
@@ -129,3 +130,60 @@ func readOrganizationID(writer http.ResponseWriter, request *http.Request) (type
 
 	return types.OrgID(organizationID), nil
 }
+
+// readClusterNames does the same as `readClusterName`, except for multiple clusters.
+func readClusterNames(writer http.ResponseWriter, request *http.Request) ([]types.ClusterName, error) {
+	clusterNamesParam, err := getRouterParam(request, "clusters")
+	if err != nil {
+		message := fmt.Sprintf("Cluster names are not provided %v", err.Error())
+		log.Println(message)
+		// See `readClusterName`.
+		responses.SendInternalServerError(writer, message)
+
+		return []types.ClusterName{}, err
+	}
+
+	clusterNamesConverted := []types.ClusterName{}
+	for _, cn := range strings.Split(",", clusterNamesParam) {
+		if _, err := uuid.Parse(cn); err != nil {
+			const message = "cluster name format is invalid"
+
+			log.Println(message)
```

Comment:
Same here

Reply:
Please look at the original functions that I used as references: readClusterName and readOrganizationID. They have the exact same behavior.